src/matcher.py

```diff
@@ -31,45 +32,68 @@ def matcher_lambda_handler(event, lambda_context):
         efs_root_location = decrypt(os.environ["ROOT_DIRECTORY"])
         environment = decrypt(os.environ["ENVIRONMENT"])
         output_queue = decrypt(os.environ["OUTPUT_QUEUE"])
+        input_queue = decrypt(os.environ["INPUT_QUEUE"])
```

Comment:
Was this env variable already available?

Reply:
It was yeah, I was surprised as well. I must have added it and forgotten to use it.